packages/ui-core/src/typography/fontloader.component.test.tsx

```diff
@@ -0,0 +1,20 @@
+import WebFont from 'webfontloader';
+import FontLoaderInstance from './fontloader.component';
```

Comment:
To keep common convention across the code I suggest to rename file just to `font-loader.ts` as `.component` could be misleading and someone could expect a `React` component in that file.